content/3_Silver/Binary_Search.mdx

```diff
@@ -894,37 +924,31 @@ functions.
 #include <bits/stdc++.h>
 using namespace std;
 
-using ll = long long;
-
-using vi = vector<int>;
-#define pb push_back
-#define rsz resize
-#define all(x) begin(x), end(x)
-#define sz(x) (int)(x).size()
-
-using pi = pair<int,int>;
-#define f first
-#define s second
-#define mp make_pair
-
-void setIO(string name = "") { // name is nonempty for USACO file I/O
-	ios_base::sync_with_stdio(0); cin.tie(0); // see Fast Input & Output
+void setIO(string name="") {  // name is nonempty for USACO file I/O
+	ios_base::sync_with_stdio(0); cin.tie(0);  // see Fast Input & Output
 	// alternatively, cin.tie(0)->sync_with_stdio(0);
-	if (sz(name)) {
-		freopen((name+".in").c_str(), "r", stdin); // see Input & Output
+	if (!name.empty()) {
+		freopen((name+".in").c_str(), "r", stdin);  // see Input & Output
 		freopen((name+".out").c_str(), "w", stdout);
 	}
 }
 
 int main() {
 	setIO("haybales");
-	int N,Q; cin >> N >> Q;
+	int N;
+	int Q;
+	cin >> N >> Q;
 	vector<int> v(N);
-	for (int i = 0; i < N; ++i) cin >> v[i];
-	sort(begin(v),end(v));
-	for (int i = 0; i < Q; ++i) {
-		int A,B; cin >> A >> B;
-		cout << upper_bound(begin(v),end(v),B)-lower_bound(begin(v),end(v),A) << "\n";
+	for (int i = 0; i < N; i++) {
+		cin >> v[i];
+	}
+	
+	sort(begin(v), end(v));
+	for (int i = 0; i < Q; i++) {
+		int A;
+		int B;
+		cin >> A >> B;
+		cout << upper_bound(begin(v), end(v), B) - lower_bound(begin(v), end(v), A) << "\n";
```

Comment:
this line is too long

Reply:
89 chars, should be fine

Reply:
uhh it's 92 characters, and since the suggested is 80 that's too long

Reply:
ok cool
how should i split it up
all the previous methods i tried made it look really weird

Reply:
does this look okay?
```cpp
		cout << upper_bound(begin(v), end(v), B) 
					- lower_bound(begin(v), end(v), A) << "\n";
```

Reply:
looks a bit sus
but better than nothing ig